Makefile

```
@@ -263,6 +263,7 @@ endif
 
 ifdef armhf
 tars += telegraf-$(tar_version)_linux_armhf.tar.gz
+tars += telegraf-$(tar_version)_freebsd_armhf.tar.gz
```

"armhf" is a debian linux term. Freebsd calls it something different.

https://www.freebsd.org/platforms/

Since we have GOARM = 7 later on, I think we want to call this armv7
```suggestion
tars += telegraf-$(tar_version)_freebsd_armv7.tar.gz
```

It's kind of a pain that each OS and distribution seems to have a different name for the different arm versions.